src/server/raft_client.rs

```diff
@@ -893,8 +905,28 @@ where
         !self.need_flush.is_empty()
     }
 
+    fn flush_full_metrics(&mut self) {
+        if self.full_stores.is_empty() {
+            return;
+        }
+
+        for id in &self.full_stores {
+            if let Some(s) = self.cache.get_mut(id) {
+                s.full = false;
+            }
+            REPORT_FAILURE_MSG_COUNTER
+                .with_label_values(&["full", &id.0.to_string()])
```

Comment:
Maybe the metric should also include the `from_store` instead of just the `to_store`.

Reply:
`from_store` is in metrics already behind the field `instance`.